### PR TITLE
Add spanish translation skeleton

### PR DESCRIPTION
**Pull request by eduardomontalvo**
Skeleton for translate the project to Spanish language

### CODE REVIEW

**Comment by i0natan** [comment]
@eduardomontalvo WOW! amazing

1. Is this Google translate or manual work?
2. How would you suggest to display a new language? 
3. Shouldn't we add different files for Spanish (i.e. not modify existing)?

cc @idori

**Comment by idori** [comment]
I don't see any standard set by github for translating repositories.
I can suggest to create a `/translations` or `/i18n` sub-folder with a ISO language code folder name for each language. 
This will make it easy to add new translations as all relative links should still work so there is no need to add a suffix/prefix to each file.

**Comment by eduardomontalvo** [comment]
@i0natan 
1. It's a manual work.
2. I think that the correct way is create a folder with translations. For example: https://github.com/jwasham/coding-interview-university and add a new link in the repo introduction for a new translation.
3. Yes, but your advices are good to know in spanish language community, after we can add a new links in Spanish with the best resources for node.

@idori 
Exactly! ISO is a good idea, but for example in Spanish, the specification contains a single code for each spanish-speak country. For example, I propose modify the folder and rename by __es__ and wrap the Spanish language with common words for all countries (without slangs or regionalisms).

**Comment by i0natan** [comment]
Manual work? wow

@eduardomontalvo @idori what should be the UX: (a) we display links in the top to different languages (b) home page stays in English and link to each section contains few more links: 'Spanish'' Chineese', etc

**Comment by eduardomontalvo** [comment]
@i0natan I think that the b is the best way. @idori what do you think?

**Comment by i0natan** [comment]
I also vote for B

@idori you're the UX master (along with other things)

**Comment by idori** [comment]
I'm actually thinking (a) is better - leave the repository as is and add a list of links for the translations sub folders at the top (i.e. a link at the top to the Spanish translation will lead to `/translations/es/readme.md`.

@eduardomontalvo I think you changed one of the original sections in your fork - section `project structure` is translated in the original folder instead of under `/translations`

**Comment by i0natan** [comment]
@eduardomontalvo @idori my Spanish friend that it's written in a very clear and professional language

We plan to publish project milestones this week (new major releases), this can be on the list as it's a big contribution

File: sections/projectstructre/breakintcomponents.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
 
 <br/><br/>
 
 
-### Blog Quote: "Scaling requires scaling of the entire application"
- From the blog MartinFowler.com
+### Cita de Blog: "El escalado requiere escalar toda la aplicación"
```
Comment (MetCoder95) [inline sections/projectstructre/breakintcomponents.md]: When it's a quote, do not translate the title of the blog, article, book, chapter of the book, etc. where the quote comes from. By convention, keep the title in the original language and only translate the content of the quote.

File: sections/projectstructre/separateexpress.md [hunk 2]
```diff
@@ -41,7 +37,5 @@ app.set('port', port);
 var server = http.createServer(app);
 
 ```
-
-
-### Example: test your API in-process using supertest (popular testing package)
+### Ejemplo: prueba tu API en proceso usanso supertest (paquete de testing popular)
```
Comment (MetCoder95) [inline sections/projectstructre/separateexpress.md]: Maybe is **usando** and not **usanso**

File: sections/projectstructre/breakintcomponents.md (hunk 1 above)
Comment (eduardomontalvo) [reply sections/projectstructre/breakintcomponents.md]: It's true! my mistake! thx 👍

**Comment by i0natan** [comment]
@idori can u add to our milestones and think how to push forward?

**Comment by i0natan** [comment]
@eduardomontalvo sorry for the delay, we're ready to kick it off and be glad if you lead it. No need to spend upfront mass effort - we may gather few collaborators and translate slowly.

Once this is done you can get your name boldly on top of the Spanish page + added to project core collaborators. LMK whether this is of interest?

**Comment by eduardomontalvo** [comment]
@i0natan I'm glad to hear that. I'm ready to work with you and the community :)

**Comment by i0natan** [comment]
@eduardomontalvo see here:
https://github.com/i0natan/nodebestpractices/issues/95

do you need help in copying the content to the new branch?

**Comment by sagirk** [comment]
@eduardomontalvo @MetCoder95 @idori @i0natan Seems this PR has remained stalled for quite some time now. What can we do to nudge it in the direction of progress and move it closer to mergeability?

**Comment by i0natan** [comment]
@sagirk The Spanish translation is now handled by @desko27 in #95 

We should assign each core member a language, I can handle Spanish

**Comment by MetCoder95** [comment]
> @eduardomontalvo @MetCoder95 @idori @i0natan Seems this PR has remained stalled for quite some time now. What can we do to nudge it in the direction of progress and move it closer to mergeability?

I think we could assign to each member who can work with Spanish a section of the text to work with it. For example, I can work with `breakincomponents.md`, and on.

**Comment by desko27** [comment]
Hey, I recently finished one of my projects and I have extra time for this. Everything is translated indeed, I'm opening new PR so we can move this forward, if you agree.

**Comment by sagirk** [comment]
>I think we could assign to each member who can work with Spanish a section of the text to work with it. For example, I can work with breakincomponents.md, and on.

Sounds good, @MetCoder95! Thanks for the suggestion. Once @i0natan and @desko27 open a PR with the [so far translated content](https://github.com/i0natan/nodebestpractices/issues/95#issuecomment-407679220), we could run with this approach.

>Hey, I recently finished one of my projects and I have extra time for this. Everything is translated indeed, I'm opening new PR so we can move this forward, if you agree.

@i0natan 👆

**Comment by i0natan** [comment]
@desko27 I shall wait for you PR, help with merging it and then we can progress with @MetCoder95 

Did I get it right?

**Comment by desko27** [comment]
@i0natan that's totally right, working on it, it's my priority right now and I'm opening a PR ASAP.

**Pull request closed without merging**